x-pack/plugins/cases/public/components/case_view/case_view_metrics.tsx

```diff
@@ -6,35 +6,280 @@
  */
```

Comment:
This file is getting too big. Consider moving some of the helpers to another file to ease the reading of the component.

Reply:
We can create a folder metrics and each metric component could be on its own file along with all the needed helpers.